library/core/src/ops/control_flow.rs

```diff
@@ -126,7 +126,7 @@ impl<B, C> const ops::FromResidual for ControlFlow<B, C> {
 }
 
 #[unstable(feature = "try_trait_v2_residual", issue = "91285")]
-impl<B, C> ops::Residual<C> for ControlFlow<B, convert::Infallible> {
+impl<B, C> const ops::Residual<C> for ControlFlow<B, convert::Infallible> {
```

Comment:
Please add individual `rustc_const_unstable` feature gates with different tracking issues.

Reply:
Sorry, I forgot to mention. This is because `const_trait_impl` is experimental and we don't want this to affect stabilization of normal features ^^

Reply:
So one feature Flag for Residual and one for Try itself?

Should I then also constify the Residual and Try impls of Result and Option?

Reply:
They could both point to #74935, `const_try`.

Reply:
the `Try` impls already use `const_convert` so maybe I should also use that for `Residual`?

Reply:
I used `const_try` (for now?)